mobile/src/store/auth/reducer.ts

```diff
@@ -38,10 +38,8 @@ const reducer = createReducer(initialState, (builder) => {
     state.dataStatus = DataStatus.FULFILLED;
     state.user = payload;
   });
-  builder.addCase(logout.fulfilled, (state) => {
-    state.dataStatus = DataStatus.FULFILLED;
-
-    return initialState;
+  builder.addCase(logout.fulfilled, () => {
+    return { ...initialState, dataStatus: DataStatus.FULFILLED };
   });
```

Comment:
Do we need set dataStatus to FULFILLED? I think if it's logout, we no longer have user and other data, so it's should be IDLE